src/main/java/pl/edu/agh/firecell/engine/algorithm/BasicAlgorithm.java

```diff
@@ -14,28 +15,46 @@ public class BasicAlgorithm implements Algorithm {
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
     private final TemperaturePropagator temperaturePropagator;
     private final FirePropagator firePropagator;
+    private final DiffusionGenerator diffusionGenerator;
     public static final double CONVECTION_COEFFICIENT = 1;
     // should be dependent on the material in the future
-    public static final double CONDUCTIVITY_COEFFICIENT = 0.2;
+    public static final double CONDUCTIVITY_COEFFICIENT_WOOD = 0.2;
+    public static final double CONDUCTIVITY_COEFFICIENT_AIR = 0.008;
+    public static final double CONDUCTIVITY_COEFFICIENT_WOOD_AIR = 0.02;
```

Comment:
These constants should be stored in hashmap 9with additional feature of sorting values inside material pair).

Reply:
As above, cover by FIRE-27